programs/dibio.c

```diff
@@ -88,6 +92,20 @@ static UTIL_time_t g_displayClock = UTIL_TIME_INITIALIZER;
 #undef MIN
 #define MIN(a,b)    ((a) < (b) ? (a) : (b))
 
+/**
+  Returns the size of a file.
+  If error returns 0. Zero filesize or error is same for us.
+  Emit warning when the file is inaccessible or zero size.
+*/
+static size_t Dib_getFileSize (const char * fileName)
+{
+    size_t fileSize = UTIL_getFileSize(fileName);
```

Comment:
minor : this code base is extremely `const` friendly. I would even call it `const` fanatic.

Any mutable variable is immediately frown upon, and should ask the question : could I do it differently ? Sometimes, there's really no other way, but in many circumstances, it's just a matter of thinking the sequence differently.

For this case, I would suggest: `if (fileSize == UTIL_FILESIZE_UNKNOWN) return 0;`. This remove the need to make `fileSize` mutable.

Reply:
Agree. Was thinking about changing it like you suggested and forgot.